doc/nrf/ug_tfm.rst

```diff
@@ -50,8 +48,8 @@ The following targets are currently supported:
 * ``nrf5340dk_nrf5340_cpuapp_ns``
 * ``nrf9160dk_nrf9160_ns``
```

Comment:
We do support them. However we support more than that.
This should really be *_nrf5340_cpuapp_ns and *_nrf9160_ns. What is the best way to document this?

Reply:
How many are there? Too many to actually be listed?

Reply:
Well, it is better to say that we support the SoCs, instead of the boards. Since the users will define their own boards most likely.
SoC support is what is important.